09_Going_Beyond/01_README.md

```diff
@@ -180,7 +180,28 @@ Like we've said this is no small task, and requires a fairly complete kernel. Ho
 
 ## Libc
 
-## Network
+TBD - by @DT
+
+## Networking
+
+Networking is another inportant feature for modern operating systems, that lets our project no longer to be confined into our emulator/machine and talks to other computers, not only the local network but the internet. 
+
+Once implemented we can write some simple clients like an irc client, or a Discord client and use it to chat and show how cool we are chatting from a client written by us on an os written by us too..
```

Comment:
haha a discord client is no small task! Unfortunately third part clients arent allowed in the terms of service, maybe you can suggest an email client or server instead?